### PR TITLE
adding missing docs

### PR DESCRIPTION
**Pull request by Tusamarco**
@rnovikovP @catalina-adam for you to review and merge

### CODE REVIEW

**Review by rnovikovP — APPROVED** [review]

File: docs/advisors/checks/innodb-strict-mode.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+
+
+## Rule
+```
+        SELECT @@global.innodb_strict_mode;
+```
+
+## Resolution
+This setting (innodb_strict_mode) is enabled by default, and we strongly recommend leaving it enabled.  This is an on-line change that can be applied with:
```
Comment (catalinaadam) [inline docs/advisors/checks/innodb-strict-mode.md:18]:
```suggestion
The **innodb_strict_mode** setting is enabled by default. Percona strongly recommends leaving this setting enabled.  
This is an online change that you can apply with:
```

**Comment by rnovikovP** [comment]
@Tusamarco, please review the suggestions and commit them if they are correct. Thanks

File: docs/advisors/checks/innodb-strict-mode.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# MySQL InnoDB strict mode not correct
+
+## Description
+When InnoDB strict mode is disabled, it's possible to have incompatible settings for
```
[the remark this answers is not in the document]
Comment (Tusamarco) [reply docs/advisors/checks/innodb-strict-mode.md:4]: @catalinaadam ok for me

[the remark this answers is not in the document]
Comment (rnovikovP) [reply docs/advisors/checks/innodb-strict-mode.md]: changed

**Review by rnovikovP — APPROVED** [review]

**Pull request merged**